pkg/actor/system.go

```diff
@@ -388,8 +388,7 @@ func (b *SystemBuilder) Build() (*System, *Router) {
 	router := NewRouter(b.name)
 	metricWorkingDurations := make([]prometheus.Counter, b.numWorker)
 	for i := range metricWorkingDurations {
-		metricWorkingDurations[i] =
-			workingDuration.WithLabelValues(b.name, strconv.Itoa(i))
+		metricWorkingDurations[i] = workingDuration.WithLabelValues(b.name, strconv.Itoa(i))
```

Comment:
Is it reformatted by linters?

Reply:
I see https://github.com/mvdan/gofumpt#added-rules

Reply:
yes, caused by linter, and sees rules cannot be disabled. - - `